Artur Zakharian/Task4.3.py

```diff
@@ -0,0 +1,9 @@
+friends = ["John", "Marta", "James"]
+enemies = ["John", "Jonathan", "Artur"]
+for i in friends:
```

Comment:
please use more understandable naming. in this case it could be 
for name in friends  
or
for person in friends
or
for friend in friends